specification/monitor/resource-manager/microsoft.insights/2017-04-01/actionGroups_API.json

```diff
@@ -464,6 +478,55 @@
                 "serviceUri"
             ]
         },
+        "ItsmReceiver": {
```

Comment:
Because these are not required properties, I don't believe this is a breaking change.

Reply:
Adding new properties - required or optional is considered a breaking change. This could result in clients unitentionally overriding the properties on the server. Imagine a resource got created with this new property. Then the GET on the resource happened from a client which did not understand this property and so ignored it. Now if the PUT happens from the same client using the resource that was obtained via GET, the value of this property on the server will be overridden.

Reply:
So it is advisable to increment the api-version for this.

Reply:
This is already released for this API version and is almost like a baseline now. In future we are planning to support patch operation for individual receiver types. We will change the version then.